tests/integration/standard/test_authentication_misconfiguration.py

```diff
@@ -14,10 +14,9 @@
 
 import unittest
 
-from tests.integration import USE_CASS_EXTERNAL, use_cluster, TestCluster
+from tests.integration import USE_CASS_EXTERNAL, use_cluster, TestCluster, SCYLLA_VERSION
 
 
-@unittest.skip('Failing with scylla')
```

Comment:
How does it different from the other tests that we skip through the patch / yaml file?